twindle-cli/cli.js

```diff
@@ -1,45 +1,57 @@
-const yargs = require("yargs");
-const { createLibraryIfNotExists } = require("./utils/library");
+const yargs = require('yargs');
+const { createLibraryIfNotExists } = require('./utils/library');
```

Comment:
@johnjacobkenny: Are we using double or single quotes for this project? or there is no standard set for this right now?
because it might happen in future that another dev comes in and making unnecessary changes again to change all single quotes to double quotes back again based on his personnel VS code settings?

Reply:
We will add prettier in future. That will take care. Let there inconsistencies for few days.

Reply:
Yeah for now this is an unintended side effect 😅

Reply:
Yeah, more like his formatter is set to default to single quotes.

Can't do anything about these until we get the Prettier config later on.